packages/core/src/utils/jwks.ts

```diff
@@ -52,3 +54,57 @@ export const exportJWK = async (key: KeyLike | Uint8Array): Promise<JWK> => {
     kid: calculateKid(jwk),
   };
 };
+
+/**
+ * Migrated from `@logto/cli` and used in `/logto-config/rotate-private-keys` API.
```

Comment:
why not directly import them from the @logto/cli. We have several places in core that import util methods from @logto/cli package.

Reply:
Ah, problem solved. Thanks